signer/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@eth-optimism/signer",
```

Comment:
I thought we were going to deprecate the changeset based release flow in favor of the new release flow

Reply:
true, i think this is a relic from when i first started work on this over a month ago.